pepper-doc/src/main/doxygen/moduleDevelopers/fitnessFeatureDesc.md

```diff
@@ -0,0 +1,156 @@
+# Fitness Check {#fitness}
+
+The fitness check ensures, that a module fits into the Pepper platform. It checks a set of features describing whether a module implements all fields and methods which should be implemented. Some of the features are required (health features) and some are optional (fitness features). When the module does not pass at least one of the health features, the overall module fitness is set to *critical*. When all health features are passed and at least one fitness is missed, the overall module fitness is *healthy*. When the module passes all features, the overall status is *fit*.
+
+To run the fitness check for an installed module run:
+\endcode
+pepper>fitness
+\endcode 
+
+## Health Features
+
+The health features are required features and ensure that a module is runnable inside Pepper.
+
+### HAS_NAME
+
+This health feature indicates whether a module's name is set. 
+When your module fails this check, call the module's super constructor and pass the module's name:
+
+\code
+public MyModule(){
+	super(MODULE_NAME);
+	...
+}
+
+\endcode
+
+### IS_READY_TO_RUN
+
+This health feature indicates whether a module is runnable on the Pepper platform. 
+To check this, the function @ref org.corpus_tools.pepper.modules.PepperModule.isReadyToStart() is called.
+When your module fails this check, it means that this method returned false.
+
+### HAS_PASSED_SELFTEST
+
+When the module provides a self test, this health feature indicates whether the self test was passed. A self test is a test setup to check whether the module is runnable within a Pepper instance (similar to a system integration test). Therefore the module needs to provide a corpus as an input and an expected corpus. The output corpus then is compared to the expected corpus.
+When your module fails this check, it means that the self test has not been passed.
+For more information on self tests check section HAS_SELFTEST.  
```

Comment:
Its in the markdown page but this section is missing from the generated HTML, at least with doxygen 1.8.11.